app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all',
+                                              'data-turbolinks-track': 'reload' %>
```

Comment:
1. coding convention:
+ nháy đôi nhé
+ indent line 7 sai: chỉ tab vào 1 tab so với line 6, chú đi hơi xa
+ sau stylesheet_link_tag chỉ 1 space hì

2. Tìm hiểu và sử dụng I18n cho code `Ruby on Rails Tutorial Sample App`
-> dùng cho các text trong toàn bộ file html nhé